requirements.yml

```diff
@@ -0,0 +1 @@
+- src: geerlingguy.certbot
```

Comment:
Could you also update README instructions, similar to hq.mkde.., so the user gets this installed during initial setup? Else it will fail half-way through provisioning with an error about "can't find geerlingguy.certbot".

Reply:
Done!